WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -487,8 +487,10 @@ private extension DashboardViewController {
     }
 
     func observeShowWebViewSheet() {
-        viewModel.$showWebViewSheet.sink { [weak self] viewModel in
-            guard let self = self else { return }
+        Publishers.CombineLatest(viewModel.$showWebViewSheet,
+                                 viewModel.$showOnboarding)
+        .sink { [weak self] viewModel, showOnboarding in
+            guard let self = self, !showOnboarding else { return }
```

Comment:
❓ is this change necessary? since this is to present a webview sheet, this doesn't conflict with the header view from my understanding 🤔